bin/p2-schedule/main.go

```diff
@@ -49,7 +49,7 @@ func main() {
 			if err != nil {
 				log.Fatalln(err)
 			}
-			path = kp.HookPath(hookType, *nodeName, manifest.ID())
+			path = kp.HookPath(hookType, manifest.ID())
```

Comment:
Consider refactoring as struct - we need more flexibility with better safety guarantees